examples/src/main/java/com/mapbox/navigation/examples/history/HistoryFilesClient.kt

```diff
@@ -0,0 +1,81 @@
+package com.mapbox.navigation.examples.history
+
+import com.google.gson.annotations.SerializedName
+import com.mapbox.navigation.core.replay.history.ReplayHistoryDTO
+import kotlin.coroutines.resume
+import kotlin.coroutines.suspendCoroutine
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import retrofit2.http.GET
+import retrofit2.http.Path
+import timber.log.Timber
+
+data class ReplayPath(
+    @SerializedName("title")
+    val title: String,
+    @SerializedName("description")
+    val description: String,
+    @SerializedName("path")
+    val path: String
+)
+
+class HistoryFilesClient {
+
+    private val retrofit = Retrofit.Builder()
+        .baseUrl("https://mapbox.github.io/driver/replay-history/")
```

Comment:
Are we 🆗 with exposing this? cc @zugaldia

Reply:
Noting that this Github page is publicly accessible, not sure about the security implications though.

Reply:
yeah, in agreement. we'll make a solution that is deliberate https://github.com/mapbox/navigation-sdks/issues/412

Reply:
OK cool, the new pages are up

https://github.com/mapbox/mapbox-navigation-history
Available history files: https://mapbox.github.io/mapbox-navigation-history/index.json
Selected files: https://mapbox.github.io/mapbox-navigation-history/navigation-history/2020-02-04-germany-highway.json

Deprecated driver reference has been taken down
https://mapbox.github.io/driver/replay-history/index.json

Reply:
cc: @1ec5 I assume they're generic enough that iOS will also be able to take advantage of this set up.